synapse/logging/opentracing.py

```diff
@@ -614,16 +639,17 @@ def get_active_span_text_map(destination=None):
     return carrier
 
 
+@only_if_tracing(ret={})
+@ensure_active_span("get the span context as a string.", ret={})
```

Comment:
Let's make `ensure_active_span` also check that opentracing is enabled, so we don't have to do both